Tribler/community/tunnel/tunnel_community.py

```diff
@@ -1161,7 +1162,10 @@ def exit_data(self, circuit_id, sock_addr, destination, data):
                     str(sock_addr), str(self.exit_sockets[circuit_id].sock_addr))
                 self.exit_sockets[circuit_id].enable()
             try:
-                self.exit_sockets[circuit_id].sendto(data, destination)
+                if destination[0] != 'fake.net':
```

Comment:
I got traceback errrors from the tcp, address not found etc. on the exit-node instance of main.py because of the DHT announcement that takes place in the tests.